src/component/splash/SplashPageOperators/SplashPageOperators.styles.tsx

```diff
@@ -72,7 +72,7 @@ export const useStyles = makeStyles()(theme => ({
         color: 'inherit',
     },
     button: {
-        background: 'white',
+        background: 'white !important',
```

Comment:
Why `!important` ?

Reply:
Just to override the MUI styles. There's probably a nice way to do this but I don't think we'll be making more changes to `SplashPageOperators`, so a quick fix made sense.